diff --git a/Validator/CocNlOnlineValidator.php b/Validator/CocNlOnlineValidator.php
index 04ca9f2..95fbac4 100644
--- a/Validator/CocNlOnlineValidator.php
+++ b/Validator/CocNlOnlineValidator.php
@@ -13,7 +13,7 @@ class CocNlOnlineValidator implements ValidatorInterface
     ) {
     }
 
-    public function validate(mixed $value, ?ComponentInterface $component = null): bool|array
+    public function validate($value, ?ComponentInterface $component = null)
     {
         if (!$this->openKvkClient->validateKvKNumber((string)$value)) {
             return [(string)__('KvK number could not be validated online')];
diff --git a/Validator/CocNlValidator.php b/Validator/CocNlValidator.php
index b2dd7ac..e894e02 100644
--- a/Validator/CocNlValidator.php
+++ b/Validator/CocNlValidator.php
@@ -11,7 +11,7 @@ class CocNlValidator implements ValidatorInterface
     ) {
     }
 
-    public function validate(mixed $value, ?ComponentInterface $component = null): bool|array
+    public function validate($value, ?ComponentInterface $component = null)
     {
         // @todo: Check if country is Dutch
 
