diff --git a/Component/Base/Map/MapRepository.php b/Component/Base/Map/MapRepository.php
index 0ea9084..3c15aa4 100644
--- a/Component/Base/Map/MapRepository.php
+++ b/Component/Base/Map/MapRepository.php
@@ -10,12 +10,12 @@ use Loki\Map\Session\MapSession;
  */
 class MapRepository extends ComponentRepository
 {
-    public function getValue(): mixed
+    public function getValue()
     {
         return $this->getSession()->getAll($this->getComponentName());
     }
 
-    public function getPropertyValue(string $name): mixed
+    public function getPropertyValue(string $name)
     {
         return $this->getSession()->get($this->getComponentName(), $name);
     }
@@ -40,7 +40,7 @@ class MapRepository extends ComponentRepository
         return $this->getSession()->getLongitude($this->getComponentName());
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (false === is_array($value)) {
             return;
diff --git a/Component/Base/Map/MapViewModel.php b/Component/Base/Map/MapViewModel.php
index feb9c70..f078f80 100644
--- a/Component/Base/Map/MapViewModel.php
+++ b/Component/Base/Map/MapViewModel.php
@@ -11,7 +11,7 @@ use Loki\Map\MapLocation\MapLocation;
  */
 class MapViewModel extends ComponentViewModel
 {
-    protected MapLocation|null|false $currentMapLocation = null;
+    protected $currentMapLocation = null;
 
     public function getJsComponentName(): ?string
     {
@@ -118,7 +118,7 @@ class MapViewModel extends ComponentViewModel
         return '';
     }
 
-    public function getCurrentMapLocation(): MapLocation|null|false
+    public function getCurrentMapLocation()
     {
         if ($this->currentMapLocation !== null) {
             return $this->currentMapLocation;
diff --git a/MapLocation/MapLocation.php b/MapLocation/MapLocation.php
index a943936..cef3a75 100644
--- a/MapLocation/MapLocation.php
+++ b/MapLocation/MapLocation.php
@@ -14,13 +14,13 @@ use Magento\Framework\View\LayoutInterface;
 class MapLocation extends DataObject implements JsonSerializable
 {
     public function __construct(
-        private readonly ImageRenderer $imageRenderer,
-        private readonly Location $location,
-        private readonly LayoutInterface $layout,
-        private readonly int $zoom = 14,
-        private readonly string $image = '',
-        private readonly bool $visible = true,
-        private readonly bool $locked = false,
+        private ImageRenderer $imageRenderer,
+        private Location $location,
+        private LayoutInterface $layout,
+        private int $zoom = 14,
+        private string $image = '',
+        private bool $visible = true,
+        private bool $locked = false,
         private bool $selectable = true,
         array $data = [],
     ) {
@@ -111,7 +111,7 @@ class MapLocation extends DataObject implements JsonSerializable
     }
 
     #[\ReturnTypeWillChange]
-    public function jsonSerialize(): mixed
+    public function jsonSerialize()
     {
         return $this->getAttributes();
     }
diff --git a/Session/MapSession.php b/Session/MapSession.php
index f8675d2..b1bd96f 100644
--- a/Session/MapSession.php
+++ b/Session/MapSession.php
@@ -21,7 +21,7 @@ class MapSession
         return [];
     }
 
-    public function get(string $prefix, string $key): mixed
+    public function get(string $prefix, string $key)
     {
         return $this->getAll($prefix)[$key] ?? null;
     }
@@ -46,7 +46,7 @@ class MapSession
         return (float)$this->get($prefix, 'latitude');
     }
 
-    public function set(string $prefix, string $key, mixed $value): void
+    public function set(string $prefix, string $key, $value): void
     {
         $lokiMaps = (array)$this->customerSession->getLokiMaps();
         $lokiMaps[$prefix][$key] = $value;
