diff --git a/ValueObject/CheckResult.php b/ValueObject/CheckResult.php
index c56bdc9..270795f 100644
--- a/ValueObject/CheckResult.php
+++ b/ValueObject/CheckResult.php
@@ -6,10 +6,10 @@ class CheckResult
 {
     public function __construct(
         private readonly string $label,
-        private readonly mixed $value,
+        private readonly $value,
         private readonly bool $valid,
         private readonly string $recommendation,
-        private readonly string|null $url = null,
+        private readonly $url = null,
     ) {
     }
 
@@ -18,7 +18,7 @@ class CheckResult
         return $this->label;
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return $this->value;
     }
diff --git a/ViewModel/SystemCheck/AddressLines.php b/ViewModel/SystemCheck/AddressLines.php
index 5df0cdd..c3b19e7 100644
--- a/ViewModel/SystemCheck/AddressLines.php
+++ b/ViewModel/SystemCheck/AddressLines.php
@@ -18,7 +18,7 @@ class AddressLines extends ScopeConfigValue
         return 'Most of the time, 3 or 4 is a good idea';
     }
 
-    public function getUrl(): string|null
+    public function getUrl()
     {
         return $this->urlFactory->create()->getUrl(
             'admin/system_config/edit',
diff --git a/ViewModel/SystemCheck/SystemCheck.php b/ViewModel/SystemCheck/SystemCheck.php
index 5b7c113..60ed0d4 100644
--- a/ViewModel/SystemCheck/SystemCheck.php
+++ b/ViewModel/SystemCheck/SystemCheck.php
@@ -26,7 +26,7 @@ abstract class SystemCheck implements SystemCheckInterface
 
     public function addCheckResult(
         string $label,
-        mixed $value,
+        $value,
         bool $valid,
         string $recommendation
     ): void {
@@ -41,7 +41,7 @@ abstract class SystemCheck implements SystemCheckInterface
         );
     }
 
-    public function getUrl(): string|null
+    public function getUrl()
     {
         return null;
     }
diff --git a/ViewModel/SystemCheck/SystemCheckInterface.php b/ViewModel/SystemCheck/SystemCheckInterface.php
index d71aa24..081f8c4 100644
--- a/ViewModel/SystemCheck/SystemCheckInterface.php
+++ b/ViewModel/SystemCheck/SystemCheckInterface.php
@@ -9,5 +9,5 @@ use LokiCheckout\CoreAdminhtml\ValueObject\CheckResultCollection;
 interface SystemCheckInterface
 {
     public function getCheckResultCollection(): CheckResultCollection;
-    public function getUrl(): string|null;
+    public function getUrl();
 }
