diff --git a/Component/Checkout/Payment/Method/AbstractMethodRepository.php b/Component/Checkout/Payment/Method/AbstractMethodRepository.php
index e90447a..baf0984 100644
--- a/Component/Checkout/Payment/Method/AbstractMethodRepository.php
+++ b/Component/Checkout/Payment/Method/AbstractMethodRepository.php
@@ -17,12 +17,12 @@ abstract class AbstractMethodRepository extends FieldRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return null;
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (!is_array($value)) {
             return;
diff --git a/Payment/Icon/IconResolver.php b/Payment/Icon/IconResolver.php
index 3216f3d..67a6ce6 100644
--- a/Payment/Icon/IconResolver.php
+++ b/Payment/Icon/IconResolver.php
@@ -21,7 +21,7 @@ class IconResolver implements IconResolverInterface
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         $paymentMethodCode = $iconResolverContext->getPaymentMethodCode();
 
diff --git a/Payment/Redirect/RedirectResolver.php b/Payment/Redirect/RedirectResolver.php
index 0f0ab2c..4ea0f99 100644
--- a/Payment/Redirect/RedirectResolver.php
+++ b/Payment/Redirect/RedirectResolver.php
@@ -8,7 +8,7 @@ use LokiCheckout\Core\Step\FinalStep\RedirectContext;
 
 class RedirectResolver implements RedirectResolverInterface
 {
-    public function resolve(RedirectContext $redirectContext): false|string
+    public function resolve(RedirectContext $redirectContext)
     {
         $paymentMethodCode = $redirectContext->getPaymentMethod()->getCode();
 
