diff --git a/Shipment/Icon/IconResolver.php b/Shipment/Icon/IconResolver.php
index 9af28c4..a6a5f1e 100644
--- a/Shipment/Icon/IconResolver.php
+++ b/Shipment/Icon/IconResolver.php
@@ -13,7 +13,7 @@ class IconResolver implements IconResolverInterface
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         if (false === $this->moduleManager->isEnabled('@todo')) {
             return false;
