diff --git a/Component/Checkout/Sidebar/Items/EditItemRepository.php b/Component/Checkout/Sidebar/Items/EditItemRepository.php
index 39a0630..b526f79 100644
--- a/Component/Checkout/Sidebar/Items/EditItemRepository.php
+++ b/Component/Checkout/Sidebar/Items/EditItemRepository.php
@@ -24,7 +24,7 @@ class EditItemRepository extends ComponentRepository
     ) {
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (!isset($value['task'])) {
             return;
@@ -106,7 +106,7 @@ class EditItemRepository extends ComponentRepository
         $this->getContext()->getMessageManager()->addNoticeMessage((string)$message);
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return null;
     }
