diff --git a/Component/Checkout/Address/SaveInAddressBook/SaveInAddressBookViewModel.php b/Component/Checkout/Address/SaveInAddressBook/SaveInAddressBookViewModel.php
index 7afbfbc7..b6fae393 100644
--- a/Component/Checkout/Address/SaveInAddressBook/SaveInAddressBookViewModel.php
+++ b/Component/Checkout/Address/SaveInAddressBook/SaveInAddressBookViewModel.php
@@ -25,7 +25,7 @@ class SaveInAddressBookViewModel extends AddressFieldViewModel
         return 'Save in address book';
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return (bool)parent::getValue();
     }
diff --git a/Component/Checkout/Step/BillingStep/BillingStepViewModel.php b/Component/Checkout/Step/BillingStep/BillingStepViewModel.php
index c98208f6..429e58e8 100644
--- a/Component/Checkout/Step/BillingStep/BillingStepViewModel.php
+++ b/Component/Checkout/Step/BillingStep/BillingStepViewModel.php
@@ -43,7 +43,7 @@ class BillingStepViewModel extends AbstractStepViewModel
         return parent::isAccessable();
     }
 
-    public function validate(): bool|array
+    public function validate()
     {
         $result = $this->getContext()->getStepValidator()->validate();
         $this->valid = true === $result;
diff --git a/Component/Checkout/Step/OneStep/OneStepViewModel.php b/Component/Checkout/Step/OneStep/OneStepViewModel.php
index b2e9b41f..074b313d 100644
--- a/Component/Checkout/Step/OneStep/OneStepViewModel.php
+++ b/Component/Checkout/Step/OneStep/OneStepViewModel.php
@@ -19,7 +19,7 @@ class OneStepViewModel extends AbstractStepViewModel
         return 'One Step';
     }
 
-    public function validate(): bool|array
+    public function validate()
     {
         $result = $this->getContext()->getStepValidator()->validate();
         $this->valid = true === $result;
diff --git a/Component/Checkout/Step/ShippingStep/ShippingStepViewModel.php b/Component/Checkout/Step/ShippingStep/ShippingStepViewModel.php
index 27ee9c8d..63f6c3c6 100644
--- a/Component/Checkout/Step/ShippingStep/ShippingStepViewModel.php
+++ b/Component/Checkout/Step/ShippingStep/ShippingStepViewModel.php
@@ -34,7 +34,7 @@ class ShippingStepViewModel extends AbstractStepViewModel
         return !(bool)$this->getContext()->getCheckoutState()->getQuote()->isVirtual();
     }
 
-    public function validate(): bool|array
+    public function validate()
     {
         $result = $this->getContext()->getStepValidator()->validate();
         $this->valid = true === $result;
diff --git a/Component/Checkout/Step/StepViewModelInterface.php b/Component/Checkout/Step/StepViewModelInterface.php
index b504b85d..a687b993 100644
--- a/Component/Checkout/Step/StepViewModelInterface.php
+++ b/Component/Checkout/Step/StepViewModelInterface.php
@@ -11,6 +11,6 @@ interface StepViewModelInterface extends ComponentViewModelInterface
     public function getLabel(): string;
     public function isEnabled(): bool;
     public function isAccessable(): bool;
-    public function validate():  bool|array;
+    public function validate();
     public function isValid(): bool;
 }
diff --git a/Test/Unit/Util/Block/CssClassTest.php b/Test/Unit/Util/Block/CssClassTest.php
index 8f464dc8..9763896c 100644
--- a/Test/Unit/Util/Block/CssClassTest.php
+++ b/Test/Unit/Util/Block/CssClassTest.php
@@ -40,7 +40,7 @@ class CssClassTest extends TestCase
         $this->assertEquals('__other scope-other class1 class2', $cssClass('class1', 'other'));
     }
 
-    private function getBlock(): Template|MockObject
+    private function getBlock()
     {
         $layout = $this->createMock(LayoutInterface::class);
         $context = $this->createMock(Context::class);
