diff --git a/Component/ProfileSwitcher/ProfileSwitcherRepository.php b/Component/ProfileSwitcher/ProfileSwitcherRepository.php
index 9058145..cf10ec4 100644
--- a/Component/ProfileSwitcher/ProfileSwitcherRepository.php
+++ b/Component/ProfileSwitcher/ProfileSwitcherRepository.php
@@ -21,12 +21,12 @@ class ProfileSwitcherRepository extends ComponentRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return (string)$this->getContext()->getCheckoutSession()->getCurrentProfile();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (empty($value)) {
             return;
