diff --git a/Component/Checkout/Shipping/DeliveryDate/DeliveryDateRepository.php b/Component/Checkout/Shipping/DeliveryDate/DeliveryDateRepository.php
index b36aa96..4f913b9 100644
--- a/Component/Checkout/Shipping/DeliveryDate/DeliveryDateRepository.php
+++ b/Component/Checkout/Shipping/DeliveryDate/DeliveryDateRepository.php
@@ -19,7 +19,7 @@ class DeliveryDateRepository extends FieldRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $quote = $this->getContext()->getCheckoutState()->getQuote();
 
@@ -35,7 +35,7 @@ class DeliveryDateRepository extends FieldRepository
         }
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (!is_array($value)) {
             return;
