diff --git a/Component/ShipperHQ/ShipperHQRepository.php b/Component/ShipperHQ/ShipperHQRepository.php
index 1f5dd65f..3e8f0cfc 100644
--- a/Component/ShipperHQ/ShipperHQRepository.php
+++ b/Component/ShipperHQ/ShipperHQRepository.php
@@ -21,13 +21,13 @@ class ShipperHQRepository extends ComponentRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return (string)$this->checkoutState->getCheckoutSession()
             ->getShqTransactionId();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (false === is_array($value)) {
             return;
