diff --git a/Component/Shipment/LocationSelect/LocationSelectRepository.php b/Component/Shipment/LocationSelect/LocationSelectRepository.php
index b1ff45c..d2ac8f9 100644
--- a/Component/Shipment/LocationSelect/LocationSelectRepository.php
+++ b/Component/Shipment/LocationSelect/LocationSelectRepository.php
@@ -23,7 +23,7 @@ class LocationSelectRepository extends MapRepository
         return (string)$this->getAmastyQuote()->getStoreId();
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return $this->getAmastyQuote()->getData();
     }
@@ -34,7 +34,7 @@ class LocationSelectRepository extends MapRepository
         return $this->quoteRepository->getByAddressId($quote->getShippingAddress()->getId());
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (false === is_array($value)) {
             return;
diff --git a/Shipment/Icon/IconResolver.php b/Shipment/Icon/IconResolver.php
index 05a6bb8..6e36e33 100644
--- a/Shipment/Icon/IconResolver.php
+++ b/Shipment/Icon/IconResolver.php
@@ -13,7 +13,7 @@ class IconResolver implements IconResolverInterface
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         if (false === $this->moduleManager->isEnabled('@todo')) {
             return false;
diff --git a/Util/AmastyQuoteRepository.php b/Util/AmastyQuoteRepository.php
index 9f64125..22130ee 100644
--- a/Util/AmastyQuoteRepository.php
+++ b/Util/AmastyQuoteRepository.php
@@ -31,7 +31,7 @@ class AmastyQuoteRepository implements ArgumentInterface
         return $locationModel;
     }
 
-    private function getQuote(): mixed
+    private function getQuote()
     {
         $quote = $this->checkoutState->getQuote();
         $shippingAddressId = $quote->getShippingAddress()->getId();
