diff --git a/Shipment/Icon/IconResolver.php b/Shipment/Icon/IconResolver.php
index 6689ce9..919547c 100644
--- a/Shipment/Icon/IconResolver.php
+++ b/Shipment/Icon/IconResolver.php
@@ -14,7 +14,7 @@ class IconResolver implements IconResolverInterface
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         if (false === $this->moduleManager->isEnabled('DHLParcel_Shipping')) {
             return false;
