diff --git a/Component/GridLayout/GridLayoutRepository.php b/Component/GridLayout/GridLayoutRepository.php
index 4cde5bc..ecb6630 100644
--- a/Component/GridLayout/GridLayoutRepository.php
+++ b/Component/GridLayout/GridLayoutRepository.php
@@ -18,7 +18,7 @@ class GridLayoutRepository extends ComponentRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $currentGridLayout = (string)$this->getContext()->getCheckoutSession()->getCurrentGridLayout();
         if (!empty($currentGridLayout)) {
@@ -28,7 +28,7 @@ class GridLayoutRepository extends ComponentRepository
         return $this->config->getGridLayoutName();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (empty($value)) {
             return;
diff --git a/Component/Theme/ThemeRepository.php b/Component/Theme/ThemeRepository.php
index 5e76979..821d549 100644
--- a/Component/Theme/ThemeRepository.php
+++ b/Component/Theme/ThemeRepository.php
@@ -18,7 +18,7 @@ class ThemeRepository extends ComponentRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $currentTheme = (string)$this->getContext()->getCheckoutSession()->getCurrentTheme();
         if (!empty($currentTheme)) {
@@ -28,7 +28,7 @@ class ThemeRepository extends ComponentRepository
         return $this->config->getTheme();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (empty($value)) {
             return;
