diff --git a/Component/Checkout/Payment/Method/Billink/BillinkRepository.php b/Component/Checkout/Payment/Method/Billink/BillinkRepository.php
index 15af89a..dd65558 100644
--- a/Component/Checkout/Payment/Method/Billink/BillinkRepository.php
+++ b/Component/Checkout/Payment/Method/Billink/BillinkRepository.php
@@ -19,12 +19,12 @@ class BillinkRepository extends FieldRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return null;
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (!is_array($value)) {
             return;
diff --git a/Component/Checkout/Payment/Method/Billink/BillinkViewModel.php b/Component/Checkout/Payment/Method/Billink/BillinkViewModel.php
index 6af775c..bb38ae0 100644
--- a/Component/Checkout/Payment/Method/Billink/BillinkViewModel.php
+++ b/Component/Checkout/Payment/Method/Billink/BillinkViewModel.php
@@ -17,11 +17,11 @@ use Magento\Store\Model\StoreManagerInterface;
 class BillinkViewModel extends CheckoutViewModel
 {
     public function __construct(
-        protected readonly MidpageConfig $midpageConfig,
-        protected readonly StoreManagerInterface $storeManager,
-        protected readonly ResolverInterface $localeResolver,
-        protected readonly UrlFactory $urlFactory,
-        protected readonly ConfigProvider $configProvider,
+        protected MidpageConfig $midpageConfig,
+        protected StoreManagerInterface $storeManager,
+        protected ResolverInterface $localeResolver,
+        protected UrlFactory $urlFactory,
+        protected ConfigProvider $configProvider,
     ) {
     }
 
diff --git a/Payment/Icon/IconResolver.php b/Payment/Icon/IconResolver.php
index d74d6d1..095878a 100644
--- a/Payment/Icon/IconResolver.php
+++ b/Payment/Icon/IconResolver.php
@@ -13,12 +13,12 @@ class IconResolver implements IconResolverInterface
     private const XML_PATH_LOGO = 'payment/%s/logo';
 
     public function __construct(
-        private readonly ModuleManager $moduleManager,
-        private readonly ScopeConfigInterface $scopeConfig,
+        private ModuleManager $moduleManager,
+        private ScopeConfigInterface $scopeConfig,
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         $paymentMethodCode = $iconResolverContext->getPaymentMethodCode();
 
diff --git a/Payment/Redirect/RedirectResolver.php b/Payment/Redirect/RedirectResolver.php
index 98f982e..25b5e1a 100644
--- a/Payment/Redirect/RedirectResolver.php
+++ b/Payment/Redirect/RedirectResolver.php
@@ -8,7 +8,7 @@ use LokiCheckout\Core\Step\FinalStep\RedirectContext;
 
 class RedirectResolver implements RedirectResolverInterface
 {
-    public function resolve(RedirectContext $redirectContext): false|string
+    public function resolve(RedirectContext $redirectContext)
     {
         if (false === str_starts_with($redirectContext->getPaymentMethod()->getCode(), 'billink_')) {
             return false;
