diff --git a/Config/Config.php b/Config/Config.php
index 3c832a7..6a262bb 100644
--- a/Config/Config.php
+++ b/Config/Config.php
@@ -14,7 +14,7 @@ class Config
     private const XML_PATH_ALGORITHM = 'loki_checkout/ab_testing/algorithm';
 
     public function __construct(
-        private readonly ScopeConfigInterface $scopeConfig
+        private ScopeConfigInterface $scopeConfig
     ) {
     }
 
diff --git a/Plugin/ConfigSkinPlugin.php b/Plugin/ConfigSkinPlugin.php
index f15904b..c43e833 100644
--- a/Plugin/ConfigSkinPlugin.php
+++ b/Plugin/ConfigSkinPlugin.php
@@ -16,9 +16,9 @@ class ConfigSkinPlugin
     private const SESSION_KEY_SKIN_INDEX = 'loki_ab_testing_skin_index';
 
     public function __construct(
-        private readonly AbTestingConfig $abTestingConfig,
-        private readonly XmlConfig $xmlConfig,
-        private readonly CheckoutSession $checkoutSession
+        private AbTestingConfig $abTestingConfig,
+        private XmlConfig $xmlConfig,
+        private CheckoutSession $checkoutSession
     ) {
     }
 
