diff --git a/Component/Checkout/Address/AccountType/AccountTypeRepository.php b/Component/Checkout/Address/AccountType/AccountTypeRepository.php
index a34bbb0..7210abd 100644
--- a/Component/Checkout/Address/AccountType/AccountTypeRepository.php
+++ b/Component/Checkout/Address/AccountType/AccountTypeRepository.php
@@ -10,7 +10,7 @@ use Loki\Field\Component\Base\Field\FieldRepository;
  */
 class AccountTypeRepository extends FieldRepository
 {
-    public function getValue(): mixed
+    public function getValue()
     {
         $addressState = $this->getContext()->getAddressStateProvider()->getAddressState($this->getComponentName());
         $accountType = (string)$addressState->getAddress()->getAccountType();
@@ -21,7 +21,7 @@ class AccountTypeRepository extends FieldRepository
         return $this->getContext()->getAccountTypeConfig()->getDefaultValue();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         $options = $this->getContext()->getAccountTypeOptions()->getOptions();
         if (false === array_key_exists($value, $options)) {
@@ -37,7 +37,7 @@ class AccountTypeRepository extends FieldRepository
         }
     }
 
-    public function getDefaultValue(): mixed
+    public function getDefaultValue()
     {
         return $this->getContext()->getAccountTypeConfig()->getDefaultValue();
     }
