diff --git a/Plugin/EnableCspConfigPlugin.php b/Plugin/EnableCspConfigPlugin.php
index adfba24..a6c671d 100644
--- a/Plugin/EnableCspConfigPlugin.php
+++ b/Plugin/EnableCspConfigPlugin.php
@@ -18,7 +18,7 @@ class EnableCspConfigPlugin
      */
     public function afterGetValue(
         ScopeConfigInterface $subject,
-        mixed $result,
+        $result,
         $path,
         $scopeType = ScopeConfigInterface::SCOPE_TYPE_DEFAULT,
         $scopeCode = null
