diff --git a/Component/Checkout/Shipping/DeliveryOptions/DeliveryOptionsRepository.php b/Component/Checkout/Shipping/DeliveryOptions/DeliveryOptionsRepository.php
index a1049bd..d60aaa1 100644
--- a/Component/Checkout/Shipping/DeliveryOptions/DeliveryOptionsRepository.php
+++ b/Component/Checkout/Shipping/DeliveryOptions/DeliveryOptionsRepository.php
@@ -67,7 +67,7 @@ class DeliveryOptionsRepository extends MapRepository
      * @throws LocalizedException
      * @throws WebapiException
      */
-    public function getValue(): mixed
+    public function getValue()
     {
         $postnlOrder = $this->getPostNLOrderByQuote();
         if ($postnlOrder->getType() === 'PG') {
@@ -115,7 +115,7 @@ class DeliveryOptionsRepository extends MapRepository
         throw new Exception('Delivery date not found');
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (!is_array($value)) {
             return;
diff --git a/Shipment/Icon/IconResolver.php b/Shipment/Icon/IconResolver.php
index b5cccac..4631289 100644
--- a/Shipment/Icon/IconResolver.php
+++ b/Shipment/Icon/IconResolver.php
@@ -14,7 +14,7 @@ class IconResolver implements IconResolverInterface
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         if (false === $this->moduleManager->isEnabled('TIG_PostNL')) {
             return false;
diff --git a/ViewModel/DeliveryDateView.php b/ViewModel/DeliveryDateView.php
index cdda07d..0aaa14b 100644
--- a/ViewModel/DeliveryDateView.php
+++ b/ViewModel/DeliveryDateView.php
@@ -14,10 +14,10 @@ use Magento\Framework\Stdlib\DateTime\Timezone;
 class DeliveryDateView implements ArgumentInterface
 {
     public function __construct(
-        private readonly CheckoutState $checkoutState,
-        private readonly OrderRepository $orderRepository,
-        private readonly LocaleResolver $localeResolver,
-        private readonly Timezone $timezone,
+        private CheckoutState $checkoutState,
+        private OrderRepository $orderRepository,
+        private LocaleResolver $localeResolver,
+        private Timezone $timezone,
         private string $timeFormat = 'H:i',
         private string $dayFormat = 'd F',
     ) {
