diff --git a/Component/Checkout/Payment/Method/AbstractMethodRepository.php b/Component/Checkout/Payment/Method/AbstractMethodRepository.php
index e90447a..baf0984 100644
--- a/Component/Checkout/Payment/Method/AbstractMethodRepository.php
+++ b/Component/Checkout/Payment/Method/AbstractMethodRepository.php
@@ -17,12 +17,12 @@ abstract class AbstractMethodRepository extends FieldRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return null;
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (!is_array($value)) {
             return;
diff --git a/Component/Checkout/Payment/Method/AbstractMethodViewModel.php b/Component/Checkout/Payment/Method/AbstractMethodViewModel.php
index 0fe3836..61575aa 100644
--- a/Component/Checkout/Payment/Method/AbstractMethodViewModel.php
+++ b/Component/Checkout/Payment/Method/AbstractMethodViewModel.php
@@ -17,11 +17,11 @@ use Magento\Store\Model\StoreManagerInterface;
 abstract class AbstractMethodViewModel extends CheckoutViewModel
 {
     public function __construct(
-        protected readonly Config $config,
-        protected readonly AdyenConfig $adyenConfig,
-        protected readonly StoreManagerInterface $storeManager,
-        protected readonly ResolverInterface $localeResolver,
-        protected readonly UrlFactory $urlFactory,
+        protected Config $config,
+        protected AdyenConfig $adyenConfig,
+        protected StoreManagerInterface $storeManager,
+        protected ResolverInterface $localeResolver,
+        protected UrlFactory $urlFactory,
     ) {
     }
 
diff --git a/Config/Config.php b/Config/Config.php
index dd7882a..dea4413 100644
--- a/Config/Config.php
+++ b/Config/Config.php
@@ -9,7 +9,7 @@ use Magento\Store\Model\ScopeInterface;
 class Config
 {
     public function __construct(
-        private readonly ScopeConfigInterface $scopeConfig,
+        private ScopeConfigInterface $scopeConfig,
     ) {
     }
 
diff --git a/Payment/Icon/IconResolver.php b/Payment/Icon/IconResolver.php
index 3216f3d..9a6d5b4 100644
--- a/Payment/Icon/IconResolver.php
+++ b/Payment/Icon/IconResolver.php
@@ -17,11 +17,11 @@ class IconResolver implements IconResolverInterface
     ];
 
     public function __construct(
-        private readonly ModuleManager $moduleManager,
+        private ModuleManager $moduleManager,
     ) {
     }
 
-    public function resolve(IconResolverContext $iconResolverContext): false|string
+    public function resolve(IconResolverContext $iconResolverContext)
     {
         $paymentMethodCode = $iconResolverContext->getPaymentMethodCode();
 
diff --git a/Payment/Redirect/RedirectResolver.php b/Payment/Redirect/RedirectResolver.php
index 0f0ab2c..4ea0f99 100644
--- a/Payment/Redirect/RedirectResolver.php
+++ b/Payment/Redirect/RedirectResolver.php
@@ -8,7 +8,7 @@ use LokiCheckout\Core\Step\FinalStep\RedirectContext;
 
 class RedirectResolver implements RedirectResolverInterface
 {
-    public function resolve(RedirectContext $redirectContext): false|string
+    public function resolve(RedirectContext $redirectContext)
     {
         $paymentMethodCode = $redirectContext->getPaymentMethod()->getCode();
 
