diff --git a/Block/Order/DeliveryDate.php b/Block/Order/DeliveryDate.php
index bd3506f..7ed6168 100644
--- a/Block/Order/DeliveryDate.php
+++ b/Block/Order/DeliveryDate.php
@@ -15,9 +15,9 @@ class DeliveryDate extends Template
 {
     public function __construct(
         Context $context,
-        private readonly DeliveryDateRepository $deliveryDateRepository,
-        private readonly Config $config,
-        private readonly Registry $registry,
+        private DeliveryDateRepository $deliveryDateRepository,
+        private Config $config,
+        private Registry $registry,
         array $data = []
     ) {
         parent::__construct($context, $data);
diff --git a/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldRepository.php b/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldRepository.php
index 853afef..748c713 100644
--- a/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldRepository.php
+++ b/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldRepository.php
@@ -18,13 +18,13 @@ use LokiCheckout\DeliveryDate\Service\ExclusionService;
 class DateFieldRepository extends FieldRepository
 {
     public function __construct(
-        private readonly ModelRepository $modelRepository,
-        private readonly ExclusionService $exclusionService,
+        private ModelRepository $modelRepository,
+        private ExclusionService $exclusionService,
         private Config $config
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $quote = $this->getContext()->getCheckoutState()->getQuote();
 
@@ -43,7 +43,7 @@ class DateFieldRepository extends FieldRepository
         }
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         $excludes = $this->exclusionService->getExclusions();
         if (in_array($value, $excludes)) {
@@ -66,7 +66,7 @@ class DateFieldRepository extends FieldRepository
         $this->modelRepository->save($deliveryDate);
     }
 
-    public function getDefaultValue(): mixed
+    public function getDefaultValue()
     {
         return date('Y-m-d');
     }
diff --git a/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldViewModel.php b/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldViewModel.php
index ba491d2..4b7f520 100644
--- a/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldViewModel.php
+++ b/Component/Checkout/Shipping/DeliveryDate/DateField/DateFieldViewModel.php
@@ -14,8 +14,8 @@ use LokiCheckout\DeliveryDate\Service\ExclusionService;
 class DateFieldViewModel extends CheckoutViewModel
 {
     public function __construct(
-        private readonly ExclusionService $exclusionService,
-        private readonly Config $config,
+        private ExclusionService $exclusionService,
+        private Config $config,
     ) {
     }
 
diff --git a/Component/Checkout/Shipping/DeliveryDate/DeliveryDateViewModel.php b/Component/Checkout/Shipping/DeliveryDate/DeliveryDateViewModel.php
index 530766c..ff85c39 100644
--- a/Component/Checkout/Shipping/DeliveryDate/DeliveryDateViewModel.php
+++ b/Component/Checkout/Shipping/DeliveryDate/DeliveryDateViewModel.php
@@ -18,10 +18,10 @@ use LokiCheckout\DeliveryDate\Repository\TimeslotRepository;
 class DeliveryDateViewModel extends CheckoutViewModel
 {
     public function __construct(
-        private readonly ExclusionRepository $exclusionRepository,
-        private readonly TimeslotRepository $timeslotRepository,
-        private readonly Config $config,
-        private readonly CheckoutState $checkoutState
+        private ExclusionRepository $exclusionRepository,
+        private TimeslotRepository $timeslotRepository,
+        private Config $config,
+        private CheckoutState $checkoutState
     ) {
     }
 
diff --git a/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldRepository.php b/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldRepository.php
index 853457e..91cf931 100644
--- a/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldRepository.php
+++ b/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldRepository.php
@@ -16,12 +16,12 @@ use LokiCheckout\DeliveryDate\Service\TimeslotService;
 class TimeslotFieldRepository extends FieldRepository
 {
     public function __construct(
-        private readonly TimeslotService $timeslotService,
-        private readonly ModelRepository $modelRepository,
+        private TimeslotService $timeslotService,
+        private ModelRepository $modelRepository,
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $quote = $this->getContext()->getCheckoutState()->getQuote();
 
@@ -39,7 +39,7 @@ class TimeslotFieldRepository extends FieldRepository
         return $value;
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (false === $this->timeslotService->isAvailableTimeslot((string)$value)) {
             $this->getContext()->getMessageManager()->addWarningMessage(__('This timeslot is not available.'));
diff --git a/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldViewModel.php b/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldViewModel.php
index 4b6dae5..ac23c98 100644
--- a/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldViewModel.php
+++ b/Component/Checkout/Shipping/DeliveryDate/TimeslotField/TimeslotFieldViewModel.php
@@ -13,7 +13,7 @@ use LokiCheckout\DeliveryDate\Config\Config;
 class TimeslotFieldViewModel extends CheckoutViewModel
 {
     public function __construct(
-        private readonly Config $config,
+        private Config $config,
     ) {
     }
 
diff --git a/Config/Config.php b/Config/Config.php
index ce9d381..e1ce3e4 100644
--- a/Config/Config.php
+++ b/Config/Config.php
@@ -22,7 +22,7 @@ class Config implements ArgumentInterface
     private const XML_PATH_HOLIDAY_API_KEY = 'loki_checkout_delivery_date/holiday/api_key';
 
     public function __construct(
-        private readonly ScopeConfigInterface $scopeConfig
+        private ScopeConfigInterface $scopeConfig
     ) {
     }
 
diff --git a/Console/Command/DeliveryDate/CreateCommand.php b/Console/Command/DeliveryDate/CreateCommand.php
index 9db8ca6..8629ba0 100644
--- a/Console/Command/DeliveryDate/CreateCommand.php
+++ b/Console/Command/DeliveryDate/CreateCommand.php
@@ -22,8 +22,8 @@ class CreateCommand extends Command
     private const OPTION_DELIVERY_TIMESPAN = 'delivery-timeslot';
 
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
-        private readonly DeliveryDateFactory $deliveryDateFactory,
+        private DeliveryDateRepository $deliveryDateRepository,
+        private DeliveryDateFactory $deliveryDateFactory,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/DeliveryDate/DeleteCommand.php b/Console/Command/DeliveryDate/DeleteCommand.php
index 63fa003..07eccc0 100644
--- a/Console/Command/DeliveryDate/DeleteCommand.php
+++ b/Console/Command/DeliveryDate/DeleteCommand.php
@@ -19,7 +19,7 @@ class DeleteCommand extends Command
     private const ARGUMENT_ENTITY_ID = 'entity-id';
 
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
+        private DeliveryDateRepository $deliveryDateRepository,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/DeliveryDate/EditCommand.php b/Console/Command/DeliveryDate/EditCommand.php
index 0b1e744..5d44aa5 100644
--- a/Console/Command/DeliveryDate/EditCommand.php
+++ b/Console/Command/DeliveryDate/EditCommand.php
@@ -23,7 +23,7 @@ class EditCommand extends Command
     private const OPTION_DELIVERY_TIMESPAN = 'delivery-timeslot';
 
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
+        private DeliveryDateRepository $deliveryDateRepository,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/DeliveryDate/ListCommand.php b/Console/Command/DeliveryDate/ListCommand.php
index ef0c572..58f00f0 100644
--- a/Console/Command/DeliveryDate/ListCommand.php
+++ b/Console/Command/DeliveryDate/ListCommand.php
@@ -21,8 +21,8 @@ class ListCommand extends Command
     private const OPTION_DATE_TO = 'date-to';
 
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
-        private readonly SearchCriteriaBuilder $searchCriteriaBuilder,
+        private DeliveryDateRepository $deliveryDateRepository,
+        private SearchCriteriaBuilder $searchCriteriaBuilder,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Exclusion/CreateCommand.php b/Console/Command/Exclusion/CreateCommand.php
index 4cd0ed5..2352c70 100644
--- a/Console/Command/Exclusion/CreateCommand.php
+++ b/Console/Command/Exclusion/CreateCommand.php
@@ -37,8 +37,8 @@ class CreateCommand extends Command
     ];
 
     public function __construct(
-        private readonly ExclusionRepository $exclusionRepository,
-        private readonly ExclusionFactory $exclusionFactory,
+        private ExclusionRepository $exclusionRepository,
+        private ExclusionFactory $exclusionFactory,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Exclusion/DeleteCommand.php b/Console/Command/Exclusion/DeleteCommand.php
index 861b5a7..507b76f 100644
--- a/Console/Command/Exclusion/DeleteCommand.php
+++ b/Console/Command/Exclusion/DeleteCommand.php
@@ -29,7 +29,7 @@ class DeleteCommand extends Command
     ];
 
     public function __construct(
-        private readonly ExclusionRepository $exclusionRepository,
+        private ExclusionRepository $exclusionRepository,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Exclusion/EditCommand.php b/Console/Command/Exclusion/EditCommand.php
index 4bfe4c6..7bb39c3 100644
--- a/Console/Command/Exclusion/EditCommand.php
+++ b/Console/Command/Exclusion/EditCommand.php
@@ -37,7 +37,7 @@ class EditCommand extends Command
     ];
 
     public function __construct(
-        private readonly ExclusionRepository $exclusionRepository,
+        private ExclusionRepository $exclusionRepository,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Exclusion/ListCommand.php b/Console/Command/Exclusion/ListCommand.php
index 2c548df..4c311b1 100644
--- a/Console/Command/Exclusion/ListCommand.php
+++ b/Console/Command/Exclusion/ListCommand.php
@@ -31,8 +31,8 @@ class ListCommand extends Command
     ];
 
     public function __construct(
-        private readonly ExclusionRepository $exclusionRepository,
-        private readonly SearchCriteriaBuilder $searchCriteriaBuilder,
+        private ExclusionRepository $exclusionRepository,
+        private SearchCriteriaBuilder $searchCriteriaBuilder,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/ImportHolidaysCommand.php b/Console/Command/ImportHolidaysCommand.php
index 4e8e89b..21b1709 100644
--- a/Console/Command/ImportHolidaysCommand.php
+++ b/Console/Command/ImportHolidaysCommand.php
@@ -21,8 +21,8 @@ class ImportHolidaysCommand extends Command
     private const OPTION_PROVIDER = 'provider';
 
     public function __construct(
-        private readonly Config $config,
-        private readonly DateProviderListing $dateProviderListing
+        private Config $config,
+        private DateProviderListing $dateProviderListing
     ) {
         parent::__construct();
     }
diff --git a/Console/Command/Timeslot/CreateCommand.php b/Console/Command/Timeslot/CreateCommand.php
index 6eefe2d..a0299d5 100644
--- a/Console/Command/Timeslot/CreateCommand.php
+++ b/Console/Command/Timeslot/CreateCommand.php
@@ -37,8 +37,8 @@ class CreateCommand extends Command
     ];
 
     public function __construct(
-        private readonly TimeslotRepository $timeslotRepository,
-        private readonly TimeslotFactory $timeslotFactory,
+        private TimeslotRepository $timeslotRepository,
+        private TimeslotFactory $timeslotFactory,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Timeslot/DeleteCommand.php b/Console/Command/Timeslot/DeleteCommand.php
index 6473572..0578322 100644
--- a/Console/Command/Timeslot/DeleteCommand.php
+++ b/Console/Command/Timeslot/DeleteCommand.php
@@ -29,7 +29,7 @@ class DeleteCommand extends Command
     ];
 
     public function __construct(
-        private readonly TimeslotRepository $timeslotRepository,
+        private TimeslotRepository $timeslotRepository,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Timeslot/EditCommand.php b/Console/Command/Timeslot/EditCommand.php
index 130bd12..4f530ec 100644
--- a/Console/Command/Timeslot/EditCommand.php
+++ b/Console/Command/Timeslot/EditCommand.php
@@ -37,7 +37,7 @@ class EditCommand extends Command
     ];
 
     public function __construct(
-        private readonly TimeslotRepository $timeslotRepository,
+        private TimeslotRepository $timeslotRepository,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Console/Command/Timeslot/ListCommand.php b/Console/Command/Timeslot/ListCommand.php
index e4f22be..399c072 100644
--- a/Console/Command/Timeslot/ListCommand.php
+++ b/Console/Command/Timeslot/ListCommand.php
@@ -31,8 +31,8 @@ class ListCommand extends Command
     ];
 
     public function __construct(
-        private readonly TimeslotRepository $timeslotRepository,
-        private readonly SearchCriteriaBuilder $searchCriteriaBuilder,
+        private TimeslotRepository $timeslotRepository,
+        private SearchCriteriaBuilder $searchCriteriaBuilder,
         ?string $name = null
     ) {
         parent::__construct($name);
diff --git a/Controller/Adminhtml/Exclusion/Delete.php b/Controller/Adminhtml/Exclusion/Delete.php
index e829509..005f099 100644
--- a/Controller/Adminhtml/Exclusion/Delete.php
+++ b/Controller/Adminhtml/Exclusion/Delete.php
@@ -15,7 +15,7 @@ class Delete extends Action
 
     public function __construct(
         Context $context,
-        private readonly ExclusionRepository $exclusionRepository
+        private ExclusionRepository $exclusionRepository
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Exclusion/Form.php b/Controller/Adminhtml/Exclusion/Form.php
index 164bf4d..8216e40 100644
--- a/Controller/Adminhtml/Exclusion/Form.php
+++ b/Controller/Adminhtml/Exclusion/Form.php
@@ -15,7 +15,7 @@ class Form extends Action
 
     public function __construct(
         Context $context,
-        private readonly PageFactory $resultPageFactory
+        private PageFactory $resultPageFactory
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Exclusion/Grid.php b/Controller/Adminhtml/Exclusion/Grid.php
index 88f7ab0..0fbd846 100644
--- a/Controller/Adminhtml/Exclusion/Grid.php
+++ b/Controller/Adminhtml/Exclusion/Grid.php
@@ -15,7 +15,7 @@ class Grid extends Action
 
     public function __construct(
         Context $context,
-        private readonly PageFactory $resultPageFactory
+        private PageFactory $resultPageFactory
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Exclusion/MassDelete.php b/Controller/Adminhtml/Exclusion/MassDelete.php
index 6bff101..472ff99 100644
--- a/Controller/Adminhtml/Exclusion/MassDelete.php
+++ b/Controller/Adminhtml/Exclusion/MassDelete.php
@@ -15,7 +15,7 @@ class MassDelete extends Action
 
     public function __construct(
         Context $context,
-        private readonly ExclusionRepository $exclusionRepository
+        private ExclusionRepository $exclusionRepository
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Holiday/Import.php b/Controller/Adminhtml/Holiday/Import.php
index be14463..827e1c5 100644
--- a/Controller/Adminhtml/Holiday/Import.php
+++ b/Controller/Adminhtml/Holiday/Import.php
@@ -17,8 +17,8 @@ class Import implements HttpPostActionInterface
     public const ADMIN_RESOURCE = 'LokiCheckout_DeliveryDate::holiday_import';
 
     public function __construct(
-        private readonly Config $config,
-        private readonly DateProviderListing $dateProviderListing,
+        private Config $config,
+        private DateProviderListing $dateProviderListing,
         private RedirectFactory $resultRedirectFactory,
         private RequestInterface $request,
         private Manager $messageManager
diff --git a/Controller/Adminhtml/Holiday/Index.php b/Controller/Adminhtml/Holiday/Index.php
index 5069526..d1c0931 100644
--- a/Controller/Adminhtml/Holiday/Index.php
+++ b/Controller/Adminhtml/Holiday/Index.php
@@ -15,7 +15,7 @@ class Index implements HttpGetActionInterface
     public const ADMIN_RESOURCE = 'LokiCheckout_DeliveryDate::holiday_import';
 
     public function __construct(
-        private readonly PageFactory $resultPageFactory
+        private PageFactory $resultPageFactory
     ) {
     }
 
diff --git a/Controller/Adminhtml/Timeslot/Delete.php b/Controller/Adminhtml/Timeslot/Delete.php
index ca894d0..5963b3a 100644
--- a/Controller/Adminhtml/Timeslot/Delete.php
+++ b/Controller/Adminhtml/Timeslot/Delete.php
@@ -15,7 +15,7 @@ class Delete extends Action
 
     public function __construct(
         Context $context,
-        private readonly TimeslotRepository $timeslotRepository
+        private TimeslotRepository $timeslotRepository
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Timeslot/Form.php b/Controller/Adminhtml/Timeslot/Form.php
index 6337197..37b9eaf 100644
--- a/Controller/Adminhtml/Timeslot/Form.php
+++ b/Controller/Adminhtml/Timeslot/Form.php
@@ -15,7 +15,7 @@ class Form extends Action
 
     public function __construct(
         Context $context,
-        private readonly PageFactory $resultPageFactory
+        private PageFactory $resultPageFactory
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Timeslot/Grid.php b/Controller/Adminhtml/Timeslot/Grid.php
index 4277c59..31f9e4e 100644
--- a/Controller/Adminhtml/Timeslot/Grid.php
+++ b/Controller/Adminhtml/Timeslot/Grid.php
@@ -15,7 +15,7 @@ class Grid extends Action
 
     public function __construct(
         Context $context,
-        private readonly PageFactory $resultPageFactory
+        private PageFactory $resultPageFactory
     ) {
         parent::__construct($context);
     }
diff --git a/Controller/Adminhtml/Timeslot/MassDelete.php b/Controller/Adminhtml/Timeslot/MassDelete.php
index 495c9f9..837b0b2 100644
--- a/Controller/Adminhtml/Timeslot/MassDelete.php
+++ b/Controller/Adminhtml/Timeslot/MassDelete.php
@@ -21,7 +21,7 @@ class MassDelete implements HttpPostActionInterface
         private RedirectFactory $resultRedirectFactory,
         private Manager $messageManager,
         private RequestInterface $request,
-        private readonly TimeslotRepository $timeslotRepository
+        private TimeslotRepository $timeslotRepository
     ) {
     }
 
diff --git a/DateProvider/HolidayApiProvider.php b/DateProvider/HolidayApiProvider.php
index eb6c3be..de0b279 100644
--- a/DateProvider/HolidayApiProvider.php
+++ b/DateProvider/HolidayApiProvider.php
@@ -16,11 +16,11 @@ class HolidayApiProvider implements DateProviderInterface
     private const API_URL = 'https://holidayapi.com/v1/holidays';
 
     public function __construct(
-        private readonly Config $config,
-        private readonly ExclusionFactory $exclusionFactory,
-        private readonly ExclusionRepository $exclusionRepository,
-        private readonly Curl $curl,
-        private readonly LoggerInterface $logger
+        private Config $config,
+        private ExclusionFactory $exclusionFactory,
+        private ExclusionRepository $exclusionRepository,
+        private Curl $curl,
+        private LoggerInterface $logger
     ) {
     }
 
diff --git a/DateProvider/OpenHolidaysApiProvider.php b/DateProvider/OpenHolidaysApiProvider.php
index 8a00779..79a1950 100644
--- a/DateProvider/OpenHolidaysApiProvider.php
+++ b/DateProvider/OpenHolidaysApiProvider.php
@@ -15,10 +15,10 @@ class OpenHolidaysApiProvider implements DateProviderInterface
     private const API_URL = 'https://openholidaysapi.org/PublicHolidays';
 
     public function __construct(
-        private readonly ExclusionFactory $exclusionFactory,
-        private readonly ExclusionRepository $exclusionRepository,
-        private readonly Curl $curl,
-        private readonly LoggerInterface $logger
+        private ExclusionFactory $exclusionFactory,
+        private ExclusionRepository $exclusionRepository,
+        private Curl $curl,
+        private LoggerInterface $logger
     ) {
     }
 
diff --git a/Observer/QuoteSubmitBeforeObserver.php b/Observer/QuoteSubmitBeforeObserver.php
index ceed578..e4b616d 100644
--- a/Observer/QuoteSubmitBeforeObserver.php
+++ b/Observer/QuoteSubmitBeforeObserver.php
@@ -12,8 +12,8 @@ use Psr\Log\LoggerInterface;
 class QuoteSubmitBeforeObserver implements ObserverInterface
 {
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
-        private readonly LoggerInterface $logger
+        private DeliveryDateRepository $deliveryDateRepository,
+        private LoggerInterface $logger
     ) {
     }
 
diff --git a/Plugin/Order/LoadDeliveryDateExtensionAttribute.php b/Plugin/Order/LoadDeliveryDateExtensionAttribute.php
index 1e80b75..22d7c39 100644
--- a/Plugin/Order/LoadDeliveryDateExtensionAttribute.php
+++ b/Plugin/Order/LoadDeliveryDateExtensionAttribute.php
@@ -12,8 +12,8 @@ use Magento\Sales\Api\OrderRepositoryInterface;
 class LoadDeliveryDateExtensionAttribute
 {
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
-        private readonly OrderExtensionFactory $orderExtensionFactory
+        private DeliveryDateRepository $deliveryDateRepository,
+        private OrderExtensionFactory $orderExtensionFactory
     ) {
     }
 
diff --git a/Plugin/Quote/LoadDeliveryDateExtensionAttribute.php b/Plugin/Quote/LoadDeliveryDateExtensionAttribute.php
index f1e396c..dd15aab 100644
--- a/Plugin/Quote/LoadDeliveryDateExtensionAttribute.php
+++ b/Plugin/Quote/LoadDeliveryDateExtensionAttribute.php
@@ -12,8 +12,8 @@ use Magento\Quote\Api\Data\CartInterface;
 class LoadDeliveryDateExtensionAttribute
 {
     public function __construct(
-        private readonly DeliveryDateRepository $deliveryDateRepository,
-        private readonly CartExtensionFactory $cartExtensionFactory
+        private DeliveryDateRepository $deliveryDateRepository,
+        private CartExtensionFactory $cartExtensionFactory
     ) {
     }
 
diff --git a/Repository/DeliveryDateRepository.php b/Repository/DeliveryDateRepository.php
index ebc6db0..2ea59d6 100644
--- a/Repository/DeliveryDateRepository.php
+++ b/Repository/DeliveryDateRepository.php
@@ -19,11 +19,11 @@ use Magento\Framework\Exception\NoSuchEntityException;
 class DeliveryDateRepository
 {
     public function __construct(
-        private readonly ResourceModel $resource,
-        private readonly ModelFactory $modelFactory,
-        private readonly CollectionFactory $collectionFactory,
-        private readonly SearchResultsFactory $searchResultsFactory,
-        private readonly CollectionProcessorInterface $collectionProcessor
+        private ResourceModel $resource,
+        private ModelFactory $modelFactory,
+        private CollectionFactory $collectionFactory,
+        private SearchResultsFactory $searchResultsFactory,
+        private CollectionProcessorInterface $collectionProcessor
     ) {
     }
 
diff --git a/Repository/ExclusionRepository.php b/Repository/ExclusionRepository.php
index ab7dd26..d135d85 100644
--- a/Repository/ExclusionRepository.php
+++ b/Repository/ExclusionRepository.php
@@ -24,11 +24,11 @@ use Magento\Framework\Api\SearchResults;
 class ExclusionRepository
 {
     public function __construct(
-        private readonly ExclusionResource $resource,
-        private readonly ExclusionFactory $exclusionFactory,
-        private readonly CollectionFactory $collectionFactory,
-        private readonly SearchResultsFactory $searchResultsFactory,
-        private readonly CollectionProcessorInterface $collectionProcessor,
+        private ExclusionResource $resource,
+        private ExclusionFactory $exclusionFactory,
+        private CollectionFactory $collectionFactory,
+        private SearchResultsFactory $searchResultsFactory,
+        private CollectionProcessorInterface $collectionProcessor,
     ) {
     }
 
diff --git a/Repository/TimeslotRepository.php b/Repository/TimeslotRepository.php
index 2d814b7..b1cc5a7 100644
--- a/Repository/TimeslotRepository.php
+++ b/Repository/TimeslotRepository.php
@@ -20,11 +20,11 @@ use Magento\Framework\Exception\NoSuchEntityException;
 class TimeslotRepository
 {
     public function __construct(
-        private readonly TimeslotResource $resource,
-        private readonly TimeslotFactory $timeslotFactory,
-        private readonly CollectionFactory $collectionFactory,
-        private readonly SearchResultsFactory $searchResultsFactory,
-        private readonly CollectionProcessorInterface $collectionProcessor
+        private TimeslotResource $resource,
+        private TimeslotFactory $timeslotFactory,
+        private CollectionFactory $collectionFactory,
+        private SearchResultsFactory $searchResultsFactory,
+        private CollectionProcessorInterface $collectionProcessor
     ) {
     }
 
diff --git a/Setup/Patch/Data/AddDefaultTimeslots.php b/Setup/Patch/Data/AddDefaultTimeslots.php
index ba9ad1f..dbf0b6b 100644
--- a/Setup/Patch/Data/AddDefaultTimeslots.php
+++ b/Setup/Patch/Data/AddDefaultTimeslots.php
@@ -16,9 +16,9 @@ class AddDefaultTimeslots implements DataPatchInterface
      * @param TimeslotFactory $timeslotFactory
      */
     public function __construct(
-        private readonly ModuleDataSetupInterface $moduleDataSetup,
-        private readonly TimeslotRepository $timeslotRepository,
-        private readonly TimeslotFactory $timeslotFactory
+        private ModuleDataSetupInterface $moduleDataSetup,
+        private TimeslotRepository $timeslotRepository,
+        private TimeslotFactory $timeslotFactory
     ) {
     }
 
