diff --git a/Component/ShipperHQ/ShipperHQRepository.php b/Component/ShipperHQ/ShipperHQRepository.php
index b1ee800b..b184aa8a 100644
--- a/Component/ShipperHQ/ShipperHQRepository.php
+++ b/Component/ShipperHQ/ShipperHQRepository.php
@@ -21,12 +21,12 @@ class ShipperHQRepository extends ComponentRepository
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         return (string)$this->checkoutState->getCheckoutSession()->getShqTransactionId();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (false === is_array($value)) {
             return;
