diff --git a/Component/GridLayout/GridLayoutRepository.php b/Component/GridLayout/GridLayoutRepository.php
index 111304d..090d860 100644
--- a/Component/GridLayout/GridLayoutRepository.php
+++ b/Component/GridLayout/GridLayoutRepository.php
@@ -13,12 +13,12 @@ use Loki\Components\Exception\RedirectException;
 class GridLayoutRepository extends ComponentRepository
 {
     public function __construct(
-        private readonly MessageManager $messageManager,
-        private readonly Config $config
+        private MessageManager $messageManager,
+        private Config $config
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $currentGridLayout = (string)$this->getContext()->getCheckoutSession()->getCurrentGridLayout();
         if (!empty($currentGridLayout)) {
@@ -28,7 +28,7 @@ class GridLayoutRepository extends ComponentRepository
         return $this->config->getGridLayoutName();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (empty($value)) {
             return;
diff --git a/Component/Skin/SkinContext.php b/Component/Skin/SkinContext.php
index 6fc2f66..f2f71ec 100644
--- a/Component/Skin/SkinContext.php
+++ b/Component/Skin/SkinContext.php
@@ -11,10 +11,10 @@ use Loki\Components\Component\ComponentContextInterface;
 class SkinContext implements ComponentContextInterface
 {
     public function __construct(
-        private readonly CheckoutSession $checkoutSession,
-        private readonly XmlConfig $xmlConfig,
-        private readonly ResponseFactory $responseFactory,
-        private readonly UrlFactory $urlFactory,
+        private CheckoutSession $checkoutSession,
+        private XmlConfig $xmlConfig,
+        private ResponseFactory $responseFactory,
+        private UrlFactory $urlFactory,
     ) {
     }
 
diff --git a/Component/Skin/SkinRepository.php b/Component/Skin/SkinRepository.php
index 4421ba0..ac5e14b 100644
--- a/Component/Skin/SkinRepository.php
+++ b/Component/Skin/SkinRepository.php
@@ -13,12 +13,12 @@ use Loki\Components\Exception\RedirectException;
 class SkinRepository extends ComponentRepository
 {
     public function __construct(
-        private readonly MessageManager $messageManager,
-        private readonly Config $config
+        private MessageManager $messageManager,
+        private Config $config
     ) {
     }
 
-    public function getValue(): mixed
+    public function getValue()
     {
         $currentSkin = (string)$this->getContext()->getCheckoutSession()->getCurrentSkin();
         if (!empty($currentSkin)) {
@@ -28,7 +28,7 @@ class SkinRepository extends ComponentRepository
         return $this->config->getSkin();
     }
 
-    public function saveValue(mixed $value): void
+    public function saveValue($value): void
     {
         if (empty($value)) {
             return;
diff --git a/Plugin/OverrideConfigFromSessionPlugin.php b/Plugin/OverrideConfigFromSessionPlugin.php
index 1238943..948ea3e 100644
--- a/Plugin/OverrideConfigFromSessionPlugin.php
+++ b/Plugin/OverrideConfigFromSessionPlugin.php
@@ -8,7 +8,7 @@ use LokiCheckout\Core\Config\Config;
 class OverrideConfigFromSessionPlugin
 {
     public function __construct(
-        private readonly CheckoutSession $checkoutSession,
+        private CheckoutSession $checkoutSession,
     ) {
     }
 
