diff --git a/Interception/Code/Generator/Interceptor.php b/Interception/Code/Generator/Interceptor.php
index 4b8685f..ecb1181 100644
--- a/Interception/Code/Generator/Interceptor.php
+++ b/Interception/Code/Generator/Interceptor.php
@@ -212,12 +212,40 @@ METHOD_BODY
         $returnTypeValue = null;
         $returnType = $method->getReturnType();
         if ($returnType) {
+            if ($returnType instanceof \ReflectionUnionType || $returnType instanceof \ReflectionIntersectionType) {
+                return $this->getReturnTypeValues($returnType, $method);
+            }
+
+            $className = $method->getDeclaringClass()->getName();
             $returnTypeValue = ($returnType->allowsNull() && $returnType->getName() !== 'mixed' ? '?' : '');
             $returnTypeValue .= ($returnType->getName() === 'self')
-                ? $this->_getFullyQualifiedClassName($method->getDeclaringClass()->getName())
+                ? $className ? '\\' . ltrim($className, '\\') : ''
                 : $returnType->getName();
         }
 
         return $returnTypeValue;
     }
+
+    /**
+     * Get return type values for Intersection|Union types
+     *
+     * @param \ReflectionIntersectionType|\ReflectionUnionType $returnType
+     * @param \ReflectionMethod $method
+     * @return string|null
+     */
+    private function getReturnTypeValues(
+        \ReflectionIntersectionType|\ReflectionUnionType $returnType,
+        \ReflectionMethod $method
+    ): ?string {
+        $returnTypeValue = [];
+        foreach ($method->getReturnType()->getTypes() as $type) {
+            $returnTypeValue[] =  $type->getName();
+        }
+
+        return implode(
+            $returnType instanceof \ReflectionUnionType ? '|' : '&',
+            $returnTypeValue
+        );
+    }
 }
+
